src/temp-server.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const pg = require('pg');
+const dotenv = require('dotenv');
+
+const { Pool } = pg;
+
+const app = express();
+
+dotenv.config();
+
+const signUp = async (connection) => {
+    const {fullname, email, password, username, telephone} = {
+        fullname: 'example', 
+        email: 'example1@email.com',
+        password: 'pwd', 
+        username: '@eric',
+        telephone: 788
+    };
+    try {
+        const result = await connection.query(`INSERT INTO users(fullname, email, password, username, telephone) VALUES (
+            '${fullname}',
+            '${email}',
+            '${password}',
+            '${username}',
+            '${telephone}'
+          ) returning *;
+        `);
+        await connection.end();
+    } catch (err) {
+        console.log('---- Error -----', err);
+    }
+};
+
```

Comment:
This is not meant to be in the server if I understand what you are trying to achieve. Let's sync on this after standup.

Reply:
@julietezekwe you are right no need this file. I have used it to confirm the database connection locally.